src/client.js

```diff
@@ -340,7 +340,7 @@ Client.prototype.names = function(data) {
 	}
 
 	client.emit("names", {
-		chan: target.chan.id,
```

Comment:
Shouldn't we keep this as-is, for consistency with the remaining of the code? While I admit `id` makes a lot more sense, everywhere else sends `chan: chan.id`:

```
➜ lounge git:(PR-217) grep -F 'chan.id' -B1 -R src/
src/plugins/irc-events/quit.js-			client.emit("users", {
src/plugins/irc-events/quit.js:				chan: chan.id
--
src/plugins/irc-events/join.js-		client.emit("users", {
src/plugins/irc-events/join.js:			chan: chan.id
--
src/plugins/irc-events/topic.js-		client.emit("topic", {
src/plugins/irc-events/topic.js:			chan: chan.id,
--
src/plugins/irc-events/message.js-
src/plugins/irc-events/message.js:		if (!self && chan.id !== client.activeChannel) {
--
src/plugins/irc-events/nick.js-			client.emit("users", {
src/plugins/irc-events/nick.js:				chan: chan.id
--
src/plugins/irc-events/kick.js-		client.emit("users", {
src/plugins/irc-events/kick.js:			chan: chan.id
--
src/plugins/irc-events/names.js-		client.emit("users", {
src/plugins/irc-events/names.js:			chan: chan.id
--
src/plugins/irc-events/part.js-			client.emit("part", {
src/plugins/irc-events/part.js:				chan: chan.id
--
src/plugins/irc-events/part.js-			client.emit("users", {
src/plugins/irc-events/part.js:				chan: chan.id
--
src/plugins/inputs/part.js-	this.emit("part", {
src/plugins/inputs/part.js:		chan: chan.id
--
src/client.js-	client.emit("more", {
src/client.js:		chan: chan.id,
--
src/client.js-		target.chan.highlight = false;
src/client.js:		this.activeChannel = target.chan.id;
--
src/client.js-	client.emit("names", {
src/client.js:		id: target.chan.id,
```